src/testsuite/vec_perf_f32.c

```diff
@@ -77,7 +78,7 @@ int timed_is_f32 (void)
 
 int timed_fpclassify_f32 (void)
 {
-#ifndef __clang__
+#ifndef PVECLIB_DISABLE_F128MATH
 // Clang does not define vec_test_data_class and friends.
```

Comment:
delete this comment?  Is this comment correct? The comment (about fpclassify) does not seem to match the ifdef (about Float128).

Reply:
OK